assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -317,6 +331,21 @@ class GettingStartedOverview extends React.Component<Props, State> {
             />
           </Box>
 
+          <Box mb={3}>
+            <label htmlFor="greeting">
+              Set an away message (will replace greeting message outside working
+              hours):
+            </label>
+            <Input
+              id="greeting"
```

Comment:
nit - let's change the `id` here and the `htmlFor` above from `"greeting"` to `"away_message"`

Reply:
whoops, a hasty copy+paste :)